classes/Wrapper/OrderWrapper.php

```diff
@@ -44,14 +43,14 @@ public function wrapOrder($order)
         $currency = new Currency((int) $order->id_currency);
 
         return [
-            'id' => (int) $order->id,
+            'transaction_id' => (int) $order->id,
             'affiliation' => Shop::isFeatureActive() ? $this->context->shop->name : Configuration::get('PS_SHOP_NAME'),
-            'revenue' => (float) $order->total_paid,
+            'value' => (float) $order->total_paid,
             'shipping' => (float) $order->total_shipping,
             'tax' => (float) $order->total_paid_tax_incl - $order->total_paid_tax_excl,
             'customer' => (int) $order->id_customer,
             'currency' => $currency->iso_code,
-            'payment_type' => (int) $order->payment,
+            'payment_type' => (string) $order->payment,
```

Comment:
This change (and others) make me thing: this is going to make a clear difference in data being recorded with ps_googleanalytics modules 4 and 5, right? We need to make people fully aware of that. Because they won't be able to compare data collected and data collected after?

Reply:
This was a bug